### ISSUE TITLE
Feature Request for large LDAP implementation

### ISSUE THREAD

**Issue by bitking** [nuxsmin/sysPass#761]
Hi,

The use case for me is the following: how can i let the whole company use syspass using AD login with their different groups (now specified in AD based on their OU as a Department number) with the least amount of administrative maintenance implementing and maintaining it.

I am thinking about the following: To make syspass more scalable for above situation in an enterprise with for example 3000 users i would like to see an option to import all AD users with their respective AD groups being created in syspass automatically or making it possible that when giving the command to import users to set them up in the groups you want them to be. Because i would never want all employee's to be in the same group, the differentation between them is the department which is specified in a different OU and different Security groups.

I am also open to any other suggestion or idea's that doesnt consume a lot of your time and would help me in succeeding :)

**Comment by nuxsmin** [comment]
Hi, LDAP group import is a tricky task, since users could be member of several groups so it actually would need to require to set the correct group manually.

If the import does create all the groups, you'd notice that some groups like printers assignment and other stuff will be imported too, so groups list would be a mess.

As I said, it needs to be implemented in a useful way and I'm thinking how to achieve it...

Thanks for the feedback!

PS: it's a great and useful feature

**Comment by bitking** [comment]
@nuxsmin Yes i am aware of that completely and not sure how to implement it. 
Its more the case of now i have to import 3000 people and assign all of them to groups, which is one hell of an administrative task, also in maintaining new people and ones that leave. 
How can we make it more sustainable sort of speak. I'll also contemplate on that a bit more.
